backoffice/backoffice/workflows/admin.py

```diff
@@ -124,3 +135,15 @@ class DecisionAdmin(BaseModelAdmin):
     @admin.display(description="action")
     def action_value(self, obj):
         return obj.action
+
+
+@admin.register(WorkflowTicket)
+class WorkflowTicketAdmin(BaseModelAdmin):
+    """
+    Admin class for WorkflowTicket model. Define get, update and delete permissions.
+    """
```

Comment:
do you have pre-commit installed?

Reply:
I thought I did but apparently I didn't, I have installed and run it now